NEWS.md

```diff
@@ -1,5 +1,14 @@
 # pkgdown (development version)
 
+* Make authors configuration more flexible: users can now
+  * Choose the roles used for filtering authors for the sidebar and footer.
+  * Choose the text before authors in the footer.
+  * Add text before and after the authors list of the authors page. (#1516)
```

Comment:
Text before and after authors in the sidebar?

Reply:
No, should this be added?